pages/about-us.tsx

```diff
@@ -1,7 +1,87 @@
 import React from "react";
+import { Button } from "../stories/Button/Button";
+import { ContactTeamCard } from "../stories/Card/ContactTeamCard";
+
+const aboutData = [
+  {
+    name: "Water Perez",
+    rol: "Frontend Developer",
+    description: `Lorem ipsum, dolor sit amet consectetur adipisicing elit. Rem
+  perferendis eius alias, illo tempore deserunt pariatur rerum minima.
+  Fugiat molestias veniam nulla, numquam recusandae asperiores fugit
+  voluptate nesciunt quidem esse!`,
+    social: [
+      { iconName: "github", url: "https://github.com/" },
+      { iconName: "linkedin", url: "https://www.linkedin.com/" },
+      { iconName: "link", url: "https://jairdev.netlify.app/" },
+      { iconName: "mail", url: "devalfredomoscoso@gmail.com" },
+      { iconName: "behance", url: "https://behance.net/" },
+    ],
+  },
+  {
+    name: "Juan Perez",
+    rol: "Frontend Developer",
+    description: `Lorem ipsum, dolor sit amet consectetur adipisicing elit. Rem
+  perferendis eius alias, illo tempore deserunt pariatur rerum minima.
+  Fugiat molestias veniam nulla, numquam recusandae asperiores fugit
+  voluptate nesciunt quidem esse!`,
+    social: [
+      { iconName: "github", url: "https://github.com/" },
+      { iconName: "linkedin", url: "https://www.linkedin.com/" },
+      { iconName: "link", url: "https://jairdev.netlify.app/" },
+    ],
+  },
+  {
+    name: "Frontend Perez",
+    rol: "Frontend Developer",
+    description: `Lorem ipsum, dolor sit amet consectetur adipisicing elit. Rem
+  perferendis eius alias, illo tempore deserunt pariatur rerum minima.
+  Fugiat molestias veniam nulla, numquam recusandae asperiores fugit
+  voluptate nesciunt quidem esse!`,
+    social: [
+      { iconName: "github", url: "https://github.com/" },
+      { iconName: "linkedin", url: "https://www.linkedin.com/" },
+      { iconName: "behance", url: "https://jairdev.netlify.app/" },
+    ],
+  },
+];
 
 const AboutUs = () => {
-  return <div>AboutUs</div>;
+  return (
+    <div className="p-6 md:p-10 max-w-[1440px]">
```

Comment:
```suggestion
    <div className="p-6 md:p-10 max-w-[1440px] bg-white h-full">
```
Hola @JairDev! Se ve genial! Also, puedes actualizar tu rama para que tengas el sidebar y agregar el `padding` que faltaria para mobile